### PR TITLE
Updated Application Access RFD.

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

File: rfd/0008-application-access.md [hunk 1]
```diff
@@ -168,6 +168,27 @@ Below are examples of the three events.
 }
 ```
 
+### FQDN
+
+By default applications are available at `appName.localProxyAddress`. This is the address most users will access an application at. An example would be `grafana.example.com`.
```
Comment (andrejtokarcik) [inline rfd/0008-application-access.md:173]: In other places the RFD uses `proxy.example.com` as the example proxy address; `grafana.proxy.example.com` may be therefore more descriptive. (Also: `acme.proxy.example.com` in the Logout section?)

**Review by andrejtokarcik — APPROVED** [review]

File: rfd/0008-application-access.md [hunk 2]
```diff
@@ -168,6 +168,27 @@ Below are examples of the three events.
...
+
+Here `localProxyAddress` is the `public_addr` field under `proxy_service` in file configuration or the name of the cluster if that field is not set.
+
+If the application should be available at a different address, administrators can use the `public_addr` field under `app_service` to override this address to a FQDN of their choosing.
+
+When accessing an application through Trusted Clusters, applications are only available at `appName.localProxyAddress`.
+
+In summary, if an application within the root cluster is being accessed.
+
+1. `appPublicDddr`
```
Comment (r0mant) [inline rfd/0008-application-access.md:183]:
```suggestion
1. `appPublicAddr`
```

File: rfd/0008-application-access.md [hunk 3]
```diff
@@ -168,6 +168,27 @@ Below are examples of the three events.
...
+2. `appName.rootProxyPublicAddr`
+3. `appName.rootClusterName`
```
Comment (r0mant) [inline rfd/0008-application-access.md:185]: This implies that the cluster name is resolvable hostname that points to the proxy as well, right?

**Review by r0mant — APPROVED** [review]

File: rfd/0008-application-access.md (hunk 3 above)
Comment (russjones) [reply rfd/0008-application-access.md:185]: Yes, it's a fallback of last resort.

**Pull request merged**